src/partials/70_section-blog-posts.html

```diff
@@ -6,70 +6,64 @@
         <div class="col"></div>
         <div class="col-auto dots">
           <ul>
-            <li><a href="#" class="active"></a></li>
-            <li><a href="#"></a></li>
-            <li><a href="#"></a></li>
+            <li style="display: none"><a href="#"></a></li>
```

Comment:
Dlaczego tak?

Reply:
Dodanie "display: none "liniowo, rozwiązuje problem wyświetlania nadmiarowych kropek. 
Jednak nadanie display none na klasę elemntu sprawia że w ogóle ich nie widać , a nasz slider nadaję takie same klasy dla ementów, nawet jeżeli nie powinny być wyświetlane.  Dla tego to wydaje się być jedyne i sensowne rozwiązanie, które działa.

Reply:
Inne rozwiązanie które mi przychodzi do głowy, to pobieranie informacjii ze slidera o ilości dostępnych "pages". I na tej podstawie ucinanie zawartości kontenera dla nav. Ale oczywiście będzie to oznaczać większą ilość kodu.

Reply:
Ilość kropek powinna być automatyczna - jeśli musisz coś w ten sposób robić to jest tzw. HACKIEM, czyli sposobem rozwiązania problemu, który nie jest dobrym rozwiązaniem, a "wytychem". Oprogramuj krokpki tak, aby ich ilosć była zgodna z treściami.